docs/framework/app-domains/build-single-file-assembly.md

```diff
@@ -80,6 +79,7 @@ csc -out:myCodeLibrary.dll -t:library myCode.cs
 ```vb
```

Comment:
Aside, I noticed a few of these compiler commanline blocks had language codes, but I left them in.
Probably good to to a find of ```vb\n\s*vbc and the same for ```csharp\n\s*csc

Reply:
The problem is that they want a specific console command for each language. Not really sure how to solve these now. If you change them to be console and not a language, then we need to indicate the language for each command.